src/components/icon/__snapshots__/icon.test.tsx.snap

```diff
@@ -1656,6 +1656,23 @@ exports[`OuiIcon props type crossInCircleEmpty is rendered 1`] = `
 </svg>
 `;
 
+exports[`OuiIcon props type crossInCircleFilled is rendered 1`] = `
```

Comment:
nice - notice that the snapshot still exists for `crossInACircleFilled`, which is what we want for backwards compatibility.

Reply:
I see what's going on right now. Much appreciated!